lib/move-sync/__tests__/move-sync.test.js

```diff
@@ -268,6 +270,23 @@ describe('moveSync()', () => {
     })
   })
 
+  describeIfWindows('> when dest parent is root', () => {
+    it('should not create parent directory', () => {
+      const src = path.join(TEST_DIR, 'a-file')
+      const dest = path.join(path.parse(TEST_DIR).root, 'another-file')
+
+      fse.moveSync(src, dest)
+
+      const contents = fs.readFileSync(dest, 'utf8')
+      const expected = /^sonic the hedgehog\r?\n$/
+      assert(contents.match(expected))
+      // this test is a bit special since
+      // we're moving to root, so we need to
+      // manually remove dest after we're done.
+      fse.removeSync(dest)
```

Comment:
Should be in an `afterEach` in this `describeIfWindows` block.

Reply:
Makes sense! I will update it then.